### PR TITLE
Add blog

### PR DESCRIPTION
**Pull request by saattrupdan**
Add a blog, along with a Munin release post.

Note that the post is still missing Mistral scores, but they should be done by tomorrow. But feel free to review the other parts, and I'll merge in when reviews pass and Mistral scores are in.

### CODE REVIEW

**Review by KennethEnevoldsen — APPROVED** [review]
Added a few fixes:
- there was an error in the dependencies
- and changes blog --> posts when on the blog page. (header is still blog)

**Comment by HLasse** [comment]
The results for `danskgpt-tiny` are extremely poor - might there be an error or is just that bad? Otherwise LGTM!

**Review by HLasse — APPROVED** [review]

**Comment by saattrupdan** [comment]
> The results for `danskgpt-tiny` are extremely poor - might there be an error or is just that bad? Otherwise LGTM!

It's just really bad it seems, but I can re-run a few of the results just to make sure.

**Comment by KennethEnevoldsen** [comment]
Er det værd at tage en af de svenske modeller med også (7b)?

File: docs/blog/posts/munin-7b-alpha.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+|-----------------------------------------	|---------------	|--------------	|-----------------	|---------------	|
+| gpt-4-0613                              	| 62.54 ± 2.87  	| 56.78 ± 2.56 	| 62.95 ± 3.63    	| 67.90 ± 2.43  	|
+| gpt-3.5-turbo-0613                      	| 56.18 ± 2.84  	| 51.73 ± 2.60 	| 54.30 ± 2.86    	| 62.52 ± 3.05  	|
+| mistralai/Mistral-7B-v0.1               	|               	|              	|                 	|               	|
+| danish-foundation-models/munin-7b-alpha 	| 21.46 ± 2.98  	| 20.53 ± 2.82 	| 14.59 ± 3.47    	| 29.25 ± 2.66  	|
+| mhenrichsen/danskgpt-tiny              	| 02.95 ± 1.18	  	| 01.77 ± 1.48 	| 02.63 ± 0.61	   	| 04.46 ± 1.46	  	|
+
+See [the full ScandEval
+leaderboard](https://scandeval.github.io/mainland-scandinavian-nlu-benchmark/) for more
+comparisons. Despite these challenges, we hope that our open approach encourages the
```
Comment (KennethEnevoldsen) [inline docs/blog/posts/munin-7b-alpha.md:34]:
```suggestion
See [the full ScandEval
leaderboard](https://scandeval.github.io/mainland-scandinavian-nlu-benchmark/) for an up-to-date
comparison. Despite these challenges, we hope that our open approach encourages the
```

Comment (KennethEnevoldsen) [reply docs/blog/posts/munin-7b-alpha.md:34]: Potentially worth noting that scores can become outdated

**Review by KennethEnevoldsen — APPROVED** [review]

**Comment by rlrs** [comment]
Måske fint at tage GPT-SW3 med hvis vi har tal på den? Jeg tilføjer også lige nogle links fx til Continual Pretraining.

**Comment by saattrupdan** [comment]
@KennethEnevoldsen, @HLasse and @rlrs: I appreciate your feedback and input, but please don't push to other people's branches directly. If you have comments then simply leave a review as normal. Thanks!

And I'll benchmark GPT-SW3-7b-v2 and re-benchmark danskgpt-tiny, and add those results in. I'm guessing we'll make this release on Monday.

**Comment by KennethEnevoldsen** [comment]
> @KennethEnevoldsen, @HLasse and @rlrs: I appreciate your feedback and input, but please don't push to other people's branches directly. If you have comments then simply leave a review as normal. Thanks!

Ahh sorry. I thought it was a service (easy to do while I had the branch open) - however, if you don't want it I will avoid it going forward.

**Comment by saattrupdan** [comment]
@KennethEnevoldsen @HLasse @rlrs I've included results for danskgpt-tiny and GPT-SW3-6.7b-v2 now, so I guess we're ready to release tomorrow - double check to see if everything is as it should be 🙂

**Review by KennethEnevoldsen — APPROVED** [review]
Looks good. Thanks for adding the Swedish models as well.

**Comment by rlrs** [comment]
Great, we just need to update the credits tomorrow.

**Pull request merged**